test_queue.py

```diff
@@ -4,6 +4,18 @@
 from queue import Queue
 
 
+@pytest.fixture(scope='function')
+def empty_queue():
```

Comment:
I sound like a broken record, but a docstring won't hurt. I know it feels like a waste of time, but you will thank yourself when you have code that you want to maintain and you're in the habit of documenting everything.